src/main.rs

```diff
@@ -1881,6 +1900,17 @@ impl<'a> NodeEx for Node<'a> {
     }
 }
 
+/// Check for STR(""), STR('')  anywhere. I am not sure if it can only appear at the end of a content line
```

Comment:
I appreciate the documentation. I need to write more of that myself :smile: 

That said, "I am not sure if it can only appear at the end of a content line" should be a separate comment and not part of the doc comment.
Could you move that?

I'd accept the PR without the change if necessary, but this just feels wrong to me.